bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -1,166 +1,438 @@
-      </form>
-      {% else %}
-          If interested, please see the  <a href="{{ release.get_bug_search_url() }}">complete list of changes</a> in this release.
       {% endif %}
-        {{ release.text|markdown|safe }}
-      </p>
+      
+      {% if release.text %}
+      <div class="title_text_wrapper_alt" >
+        <span class="release_details_alt">
+          <span class="version_wrapper_alt">
+            <h3>
+              {{ _('{version}')|f(version=release.version) }}
+            </h3>
+          </span>
+          <br>
+          <span class ="details_wrapper_alt">
+            <h2>
+              {{ _('{date}')|f(date=release.release_date|l10n_format_date, version=release.version) }} {{ _('{channel}')|f(channel=release.channel) }} {{ _('Channel') }}
+            </h2>
+          </span>
+        </span>
+        <span class="thank_wrapper">
+          {{ release.text|markdown|safe }} 
+        </span>
+        
+      </div>
+      {% endif %}
+      
+      {% if not release.text %}
+      <div class="title_text_wrapper" >
+        <span class="release_details">
+          <span class="version_wrapper">
+            <h3>
+              {{ _('{version}')|f(version=release.version) }}
+            </h3>
+          </span>
+          <span class ="details_wrapper">
+            <h2>
+              {{ _('{date}')|f(date=release.release_date|l10n_format_date, version=release.version) }} {{ _('{channel}')|f(channel=release.channel) }} Channel
+            </h2>
+          </span>
+              </span>
+      </div>
+      
+      
+      {% endif %}
+      {% endblock %}
+      {% block platform_switch %}
+      {% endblock %}
+      {% block notes_heading_secondary %}
     </header>
-  {% endblock %}
-
-    <div class="main-column">
+    {% endblock %}
+    <div class="notes-column">
       {% if new_features %}
       <section class="notes-section" id="new-features">
-        <h3>{{ _('What’s New') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in new_features %}
-          <li {% if not note.tag %}class="untagged"{% endif %} id="note-{{ note.id }}">
-            {% if note.tag %}
-              <b class="tag tag-{{ note.tag.lower() }}">{{ note.tag }}</b>
+        <div class="section-wrapper">
+          {% for note in new_features if note.tag =="New" %}
+          {% if loop.first %}
+          <span class="label_wrapper">
+            <p class="note_label">
+              &nbsp;
+              <i class="fa fa-star">
+              </i>
+              {{ _('new') }} 
+            </p>
+          </span>
+          {% endif %}
+          <span class="note_wrapper">
+            <ul class="section-items tagged">
+              <li {% if not note.tag %}class="untagged"{% endif %} {% if loop.index % 2 != 0 %}class = "gray_note" {% endif %} {% if loop.index % 2 == 0 %}class = "blue_note" {% endif %} id="note-{{ note.id }}">
```

Comment:
As @craigcook stated elsewhere, please clean up the spaces between attributes, equal signs, and values. All should look like `attr="value"`, not `attr = "value"` or `attr ="value"`.